protobuf_deps.bzl

```diff
@@ -12,6 +12,7 @@ PROTOBUF_MAVEN_ARTIFACTS = [
     "com.google.guava:guava-testlib:31.1-jre",
     "com.google.truth:truth:1.1.2",
     "junit:junit:4.13.2",
+    "org.jruby:jruby-complete:9.2.20.1",
```

Comment:
This shouldn't be necessary.  This forces all our downstream users to depend on a pinned version of jruby.  rules_ruby can be used to avoid this and always use a consistent version